### PR TITLE
Feature/clean demo

### PR DESCRIPTION
**Pull request by Le-Caignec**

### CODE REVIEW

File: demo/node/index.js [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
+
+  const web3Mail = new IExecWeb3Mail(ethProvider);
+
+  web3Mail.fetchMyContacts().then((contacts) => {
+    console.log('contacts', contacts);
+  });
 };
 
-main();
+test();
```
Comment (abbesBenayache) [inline demo/node/index.js:18]: import getWeb3Provider from '@iexec/web3mail';

voici une suggestion : 

import { IExecWeb3Mail, getWeb3Provider } from '@iexec/web3mail';
import { Wallet } from 'ethers';

const test = async () => {
  const ethProvider = getWeb3Provider(Wallet.createRandom().privateKey);

  const web3Mail = new IExecWeb3Mail(ethProvider);

  web3Mail.fetchMyContacts().then((contacts) => {
    console.log('contacts', contacts);
  });
};

test();

File: demo/ts-node/index.ts [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
+
+  const web3Mail = new IExecWeb3Mail(ethProvider);
+
+  web3Mail.fetchMyContacts().then((contacts) => {
+    console.log('contacts', contacts);
+  });
 };
 
-main();
+test();
```
Comment (abbesBenayache) [inline demo/ts-node/index.ts:18]: import getWeb3Provider from '@iexec/web3mail';

voici une suggestion :

import { IExecWeb3Mail, getWeb3Provider } from '@iexec/web3mail';
import { Wallet } from 'ethers';

const test = async () => {
  const ethProvider = getWeb3Provider(Wallet.createRandom().privateKey);

  const web3Mail = new IExecWeb3Mail(ethProvider);

  web3Mail.fetchMyContacts().then((contacts) => {
    console.log('contacts', contacts);
  });
};

test();

File: demo/README.md [hunk 3]
```diff
@@ -1,9 +1,17 @@
 # Demo
 
-@iexec/dataprotector demos
+basic `@iexec/web3mail` demos using various environment and bundlers.
+
+the demo creates a dummy protected data.
```
[the remark this answers is not in the document]
Comment (Le-Caignec) [reply demo/README.md]: Commit: 45420913ffdd7463ac7e11d14a47fbe6a3578846

File: demo/README.md [hunk 4]
```diff
@@ -22,3 +30,10 @@ run the demo
 ```sh
 npm start
 ```
+
+**NB:** for browser demos
+
+- you will need an ethereum wallet connected to [iexec sidechain](https://chainlist.org/chain/134)
+- click the `TEST` button to start the protected data creation
```
[the remark this answers is not in the document]
Comment (Le-Caignec) [reply demo/README.md]: Commit: 45420913ffdd7463ac7e11d14a47fbe6a3578846

File: demo/create-react-app-js/package.json [hunk 5]
```diff
@@ -3,14 +3,14 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@iexec/dataprotector": "^0.1.0",
```
[the remark this answers is not in the document]
Comment (Le-Caignec) [reply demo/create-react-app-js/package.json]: Commit: 887c66bcb811515b4e619fb89d1df579c1bb4605

**Review by abbesBenayache — APPROVED** [review]

**Pull request merged**